mitodl/micromasters
# Add template for MM Program certificate

#### What are the relevant tickets?
Fix #3519

#### What's this PR do?
Add cms model for `ProgramCertificateSignatories`, add a view for program certificate.

#### How should this be manually tested?
create a program certificate 
```from django.contrib.auth.models import User
user = User.objects.filter(username='username')[0]

from courses.models import *
from grades.models import *
from grades.factories import *

program = Program.objects.filter(title='Digital Learning')[0]
for c in program.course_set.all():

    course_run = c.courserun_set.all().first()

    ### Create grade, complete status, and certificate
    final_grade = FinalGradeFactory.create(user=user, course_run=course_run, passed=True, status='complete', grade=0.8)
    CourseRunGradingStatus.objects.create(course_run=course_run, status='complete')
    cert = MicromastersCourseCertificate.objects.create(final_grade=final_grade)
```
Once all the courses in the program have a certificate, the program certificate should get created automatically.
In the cms admin create `ProgramCertificateSignatories` for the program. And then you should be able to view the certificate.
